package.json

```diff
@@ -149,7 +149,7 @@
     "@types/gulp-sourcemaps": "0.0.35",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30309&commitId=ca57a0479266a6d84df968fce6c20b914cee3c98&env=prod&clientType=FLUENTUI)


  <details><summary><b>Calendar Compat</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Calendar Compat.Custom icons.default.chromium.png|0| Added|
|Calendar Compat.Default - RTL.default.chromium.png|0| Added|
|Calendar Compat.Default.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>